content/en/docs/reference/glossary/service.md

```diff
@@ -11,8 +11,10 @@ tags:
 - fundamental
 - core-object
 ---
-An abstract way to expose an application running on a set of {{< glossary_tooltip text="Pods" term_id="pod" >}} as a network service.
+A method for exposing an network application that is running as one or more {{< glossary_tooltip text="Pods" term_id="pod" >}} in your cluster.
```

Comment:
I'm not clear on the use of 'an network application'.
running as one or more Pods in your cluster adds value